src/identity-handler.js

```diff
@@ -23,16 +23,16 @@ var identityHandler = {
     onUserIdentified: function(mParticleUser) {
 
     },
-    onIdentifyCompleted: function(mParticleUser, identityApiRequest) {
+    onIdentifyComplete: function(mParticleUser, identityApiRequest) {
```

Comment:
looks like in the wrapper i have the past tense `onIdentifyCompleted`, whereas login/logout/modify are all the present tense `Complete`. So in addition to this change here, I'll need to make the update here as well:

https://github.com/mparticle-integrations/mparticle-javascript-web-kit-wrapper/blob/master/index.js#L209